analyzer/codechecker_analyzer/env.py

```diff
@@ -144,3 +144,10 @@ def get_binary_in_path(basename_list, versioning_pattern, env):
 
         # Return the one earliest in PATH.
         return files[0]
+
+def get_analyzer_from_path():
+    """ Return True if ANALYZERS_FROM_PATH environment variable is set. """
+    analyzers_from_path = os.environ.get('ANALYZERS_FROM_PATH', '').lower()
```

Comment:
Many times if a tool uses multiple environment variables then these are prefixed with the common name. I think this is a good convention, since this way it is less probable that the name interferes with an other tool's environment variable. In the logger we used `CC_` prefix and I think we should continue use it for our variables in the future too.